test/format/__snapshots__/index.spec.ts.snap

```diff
@@ -243,8 +243,6 @@ Traceback:
 Matanuska BASIC: v0.0.1
 swc: v1.7.26
 vite: v1.7.26
-Node.js: NODE_VERSION
```

Comment:
Potentially worth adding this back in:

1. Currently, the Node.js entry point is still the primary way of using `matbas`, and can't be removed until the QT app is fully operational.
2. Even when we switch to C++, Node.js will still be used in the build.